bootstrap.go

```diff
@@ -401,6 +407,48 @@ func upgradeToVer11(s Session) {
 	mustExecute(s, "UPDATE mysql.user SET References_priv='Y'")
 }
 
+func upgradeToVer12(s Session) {
+	s.Execute("BEGIN")
+	sql := "SELECT user, host, password FROM mysql.user WHERE password != ''"
+	rs, err := s.Execute(sql)
+	if err != nil {
+		log.Fatal(err)
+		return
+	}
+	r := rs[0]
+	sqls := make([]string, 1)
+	defer r.Close()
+	row, err := r.Next()
+	for err == nil && row != nil {
+		user := row.Data[0].GetString()
+		host := row.Data[1].GetString()
+		pass := row.Data[2].GetString()
+		newpass, err := util.OldPasswordUpgrade(pass)
```

Comment:
i think leave this function in util will make bootstrap concise although it is just used in bootstrap.